keda/templates/12-keda-deployment.yaml

```diff
@@ -48,8 +48,20 @@ spec:
           command:
           - "/keda"
           args:
-          - "-zap-log-level={{ .Values.logging.operator.level }}"
+          - --enable-leader-election
+          - "--zap-log-level={{ .Values.logging.operator.level }}"
+          - --zap-encoder=console
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8081
```

Comment:
@zroubalik I presume we can hardcode these ports since it's only used by us or should we parameterize them?

Reply:
@tomkerkhove yes, we don't have to parametrize :)